README.org

```diff
@@ -8,13 +8,41 @@ The Scarf CLI is a set of tools aimed at empowering users and maintainers of ope
 Each of the Scarf commands uses the general abstractions provided by Nomia to identify their resources. For example, when adding a package to an environment with the Environment Manager, you would specify the package to add and the environment to add it to by name, using the common Nomia conventions for string representations of names. This provides a unified interface to specified disparate areas of software distribution and development. Now they can be seamlessly combined and efficiently realized, while also enabling integration with any other system that offers Nomia resources.
 * Environment Manager
 The Scarf Environment Manager is used to manage the environments for user sessions, development workflows, and automated systems. It is similar to a package manager, except in that it treats environments as first-class resources to be manipulated, combined, transferred, etc.
+
+* Get started
+
+** Before you begin
+
+To install and use the Scarf CLI, you must have [[https://nixos.org/download.html][Nix]] and [[https://git-scm.com/downloads][Git]] installed.
+
+** Install
+
+1. Clone this repository and switch to it. Run ~git clone https://github.com/scarf-sh/scarf.git && cd scarf~.
+2. Run ~./install~.
+
+** Try it
+
+You can test that Scarf works by creating and installing a package in a Scarf environment. Try this:
+
+1. Install Nix's Node.js package into your default user environment. Run ~scarf env add nixpkgs:node~.
+2. Compare your system with the Scarf environment. Run these commands:
+   
+   #+BEGIN_SRC sh
```

Comment:
This is my first time out with org-mode and it looks kinda… off to me. I don't know if that's my fault, GitHub, or org-mode's. I don't think it much matters now, but this might present barriers to contribution later on (org-mode is not widely used for docs). We might want to look at related projects (such as Nix) and find out what formats they use and adopt one of those instead.

Reply:
:+1: Nix used to use docbook and I think uses markdown now, I don't love markdown for docs with any significant structure but it's fine for READMEs.

What's your preference for structured docs?

Reply:
I usually prefer to take cues from the community (e.g., if this were a Python project, then we'd I'd suggest reStructuredText). But if Nix used docbook, [AsciiDoc](https://asciidoctor.org/) might be a great fit. It has docbook semantics in lightweight markup. This is what it looks like (roughly, with a quick and dirty conversion with pandoc): https://github.com/ddbeck/scarf/blame/ascii-doc-demo/README.adoc

Reply:
I don't really think there is a community here yet, if there is probably Rust would be where to take cues. I'll take a look at AsciiDoc, thanks!